src/ResourceManager/Automation/Commands.Automation/Cmdlet/NewAzureAutomationModule.cs

```diff
@@ -39,6 +39,7 @@ public class NewAzureAutomationModule : AzureAutomationBaseCmdlet
         /// </summary>
         [Parameter(Position = 3, Mandatory = true, ValueFromPipelineByPropertyName = true,
             HelpMessage = "The url to a module zip package.")]
+        [Alias("ContentLinkUri")]
```

Comment:
@vrdmr @safeermohammed if we swap the parameter name and its alias, it won't be a breaking change. Existing scripts will be able to continue to use `ContentLink`; the only difference is when people tab through the possible parameters (as well as look at the docs for the cmdlet), they will see `ContentLinkUri`, which I think is a better indication of what this parameter is. The use of `ContentLink` is not lost in this swap.